service-registry/src/main/java/org/apache/servicecomb/serviceregistry/client/http/ServiceRegistryClientImpl.java

```diff
@@ -346,19 +346,26 @@ public boolean registerSchema(String microserviceId, String schemaId, String sch
 
   @Override
   public String getSchema(String microserviceId, String schemaId) {
+    return doGetSchema(microserviceId, schemaId, false);
+  }
+
+  private String doGetSchema(String microserviceId, String schemaId, boolean isAggregatedServiceCenter) {
```

Comment:
not "isAggregatedServiceCenter"
target is ServiceCenter Aggregator:
  1.for register flow, global=false
  2.for find instance flow, global=true
so the variable name better to be: global

Reply:
ok, i will change it